styles.css

```diff
@@ -68,8 +68,8 @@ nav {
 .fixed {
   position: fixed;
   display: block !important;
-  margin-right: 10px;
-  width: 100px;
+  right: 28px;
+  top: 10px;
 }
 
 .open {
```

Comment:
### Your Modal Window
![scrollable](https://user-images.githubusercontent.com/93039359/188741373-f3f06ef7-670f-4d6b-8467-5d406d667aa8.png)

- [ ] Your modal window is not showing all the information because you didn't make it scrollable please make it scrollable so where it won't fit then the user will be able to scroll on your project cards :+1: